assets/stylesheets/guide.css

```diff
@@ -45,12 +60,13 @@
 }
 
 .guide-list .guide .guide-intro {
-    padding: 6px 0;
+    padding: 0 0 6px 0;
 }
 
 .guide-list .guide .guide-tag-list {
     font-size: 16px;
     font-weight: 400;
+    display: none;
```

Comment:
this temporarily removes the tag list. 

Currently: 

![CleanShot 2024-10-04 at 09 52 36@2x](https://github.com/user-attachments/assets/59ffd1e0-7d13-4080-b01a-3c6fe2cd5df0)


With this PR: 

![CleanShot 2024-10-04 at 09 52 46@2x](https://github.com/user-attachments/assets/270b86c9-bd47-4755-9d41-b7d19b25bf60)